### PR TITLE
fixed node and cookie regex to be more inclusive, more whitespace

### PR DESCRIPTION
**Pull request by joedevivo**
@evanmcc found an issue with the egrep regex we were using on OS X 10.7.5. This replacement works on OS X 10.8.4 and 10.7.5 and takes into account unlimited whitespace

### CODE REVIEW

File: priv/base/env.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ echoerr() { echo "$@" 1>&2; }
 # Extract the target node name from node.args
 NAME_ARG=`egrep '^\-name' $RUNNER_ETC_DIR/vm.args 2> /dev/null`
 if [ -z "$NAME_ARG" ]; then
-    NODENAME=`egrep '^nodename\s?=\s?' $RUNNER_ETC_DIR/{{cuttlefish_conf}} 2> /dev/null | cut -d = -f 2`
+    NODENAME=`egrep 'nodename[ \t]*=[ \t]*' dev/dev1/etc/riak.confegrep '^nodename\s?=\s?' $RUNNER_ETC_DIR/{{cuttlefish_conf}} 2> /dev/null | cut -d = -f 2`
```
Comment (jaredmorrow) [inline priv/base/env.sh]: Looks like `dev/dev1/etc/riak.confegrep` snuck into the change when it shouldn't have.

Comment (joedevivo) [inline priv/base/env.sh]: oops!

**Comment by joedevivo** [comment]
@jaredmorrow fixed.

**Comment by jaredmorrow** [comment]
+1, tested on Solaris & FreeBSD

**Pull request merged**